src/mpi/comm/comm_split_type_nbhd.c

```diff
@@ -101,8 +101,10 @@ int MPIR_Comm_split_type_neighborhood(MPIR_Comm * comm_ptr, int split_type, int
 int MPIR_Comm_split_type_nbhd_common_dir(MPIR_Comm * user_comm_ptr, int key, const char *hintval,
                                          MPIR_Comm ** newcomm_ptr)
 {
+#ifndef HAVE_ROMIO
+    return MPI_SUCCESS;
```

Comment:
Do we need to set `*newcomm_ptr` to something here to prevent the caller from possibly using some uninitialized value? I see the only call site already initializes to `NULL`, so I guess it's not an issue currently. Still might be good to add for safety later.

Reply:
Added `*newcomm_ptr = NULL;`